internal/LibraryManangement/AdminManagement/AdminUserManangement/model/model.go

```diff
@@ -0,0 +1,104 @@
+package adminlibraryusermanangement
+
+import (
+	"time"
+
+	developertraining_v1 "github.com/vendasta/generated-protos-go/developer_training/v1"
+	"github.com/vendasta/gosdks/config"
+	vstore "github.com/vendasta/vstore/vstore/sdks/go/v1"
+)
+
+type AdminLibraryUserManangement struct {
+	Email_ID string    `vstore:"Email_ID"`
```

Comment:
I see that tables are not yet created. It is better to keep the vstore field names in lowercase.
```suggestion
	Email_ID string    `vstore:"email_id"`
```